Zotero/Scenes/Detail/Annotation Popover/Views/AnnotationViewController.swift

```diff
@@ -311,13 +311,14 @@ final class AnnotationViewController: UIViewController {
         }
 
         if editability != .notEditable {
+            var configuration = UIButton.Configuration.plain()
+            configuration.attributedTitle = AttributedString(L10n.Pdf.AnnotationPopover.delete, attributes: AttributeContainer([.font: UIFont.preferredFont(forTextStyle: .body)]))
+            configuration.baseForegroundColor = .red
+            configuration.contentInsets = NSDirectionalEdgeInsets(top: 12, leading: 0, bottom: 12, trailing: 0)
```

Comment:
Top inset was `11` before

Reply:
I changed some insets to fix issues. If I used the same insets with this API, results would be different, so I had to accommodate.